websites/C/Crave/presence.ts

```diff
@@ -0,0 +1,67 @@
+const presence = new Presence({
+		clientId: "1001288215388495953",
+	}),
+	browsingStamp = Date.now();
+
+function $(selector: string): HTMLElement {
+	return document.querySelector(selector);
+}
```

Comment:
Please do not use a sudo function like so, you can just explicitly write the querySelector at each reference